fixtures/pages/fr/fr.11tydata.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+const i18n = require("eleventy-plugin-i18n-gettext");
+const { getLangDir } = require("rtl-detect");
+const path = require("path");
+
+module.exports = () => {
+    let locale = path.basename(__dirname);
+    return i18n.enhance11tydata({}, locale, getLangDir(locale));
+};
```

Comment:
Maybe this is a bit much, but I'm wondering if you can refactor much of this out into a helper so that when we add all these 11tydata.js files you can just re-export it. 

Something like:

```javascript
"use strict"

export {default} from 'locale-helper';
```

see: https://stackoverflow.com/a/34445175

However, the above would require you to figure out how to properly determine the correct directory. So it might be easier to require that as a passed in argument and return the value. 

```javascript
"use strict";

const localeHelper = require("locale-helper");

module.exports = () => {
    return localeHelper(__dirname);
};
```

Maybe it doesn't save much but just a thought. I guess another option would be to just include the snippet in the docs and ask integrators to copy and paste that with modifications as needed.